bin/cron/mysql-metrics

```diff
@@ -0,0 +1,359 @@
+Com_show_databases
+Com_show_engine_logs
+Com_show_engine_mutex
+Com_show_engine_status
+Com_show_errors
+Com_show_events
+Com_show_fields
+Com_show_function_code
+Com_show_function_status
+Com_show_grants
+Com_show_keys
+Com_show_master_status
+Com_show_ndb_status
+Com_show_new_master
+Com_show_open_tables
+Com_show_plugins
+Com_show_privileges
+Com_show_procedure_code
+Com_show_procedure_status
+Com_show_processlist
+Com_show_profile
+Com_show_profiles
+Com_show_relaylog_events
+Com_show_slave_hosts
+Com_show_slave_status
+Com_show_status
+Com_show_storage_engines
+Com_show_table_status
+Com_show_tables
+Com_show_triggers
+Com_show_variables
+Com_show_warnings
+Com_signal
+Com_slave_start
+Com_slave_stop
+Com_stmt_close
+Com_stmt_execute
+Com_stmt_fetch
+Com_stmt_prepare
+Com_stmt_reprepare
+Com_stmt_reset
+Com_stmt_send_long_data
+Com_truncate
+Com_uninstall_plugin
+Com_unlock_tables
+Com_update
+Com_update_multi
+Com_xa_commit
+Com_xa_end
+Com_xa_prepare
+Com_xa_recover
+Com_xa_rollback
+Com_xa_start
+Connection_errors_accept
+Connection_errors_internal
+Connection_errors_max_connections
+Connection_errors_peer_addr
+Connection_errors_select
+Connection_errors_tcpwrap
+Connections
+Created_tmp_disk_tables
+Created_tmp_files
+Created_tmp_tables
+Firewall_access_denied
+Firewall_access_granted
+Firewall_cached_entries
+Flush_commands
+Handler_commit
+Handler_delete
+Handler_discover
+Handler_external_lock
+Handler_mrr_init
+Handler_prepare
+Handler_read_first
+Handler_read_key
+Handler_read_last
+Handler_read_next
+Handler_read_prev
+Handler_read_rnd
+Handler_read_rnd_next
+Handler_rollback
+Handler_savepoint
+Handler_savepoint_rollback
+Handler_update
+Handler_write
+Innodb_buffer_pool_read_ahead
+Innodb_buffer_pool_read_ahead_evicted
+Innodb_buffer_pool_read_ahead_rnd
+Innodb_buffer_pool_read_requests
+Innodb_buffer_pool_reads
+Innodb_buffer_pool_wait_free
+Innodb_buffer_pool_write_requests
+Innodb_data_fsyncs
+Innodb_data_read
+Innodb_data_reads
+Innodb_data_writes
+Innodb_data_written
+Innodb_dblwr_pages_written
+Innodb_dblwr_writes
+Innodb_log_waits
+Innodb_log_write_requests
+Innodb_log_writes
+Innodb_os_log_fsyncs
+Innodb_os_log_written
+Innodb_pages_created
+Innodb_pages_read
+Innodb_pages_written
+Innodb_row_lock_time
+Innodb_row_lock_waits
+Innodb_rows_deleted
+Innodb_rows_inserted
+Innodb_rows_read
+Innodb_rows_updated
+Innodb_truncated_status_writes
+Key_read_requests
+Key_reads
+Key_write_requests
+Key_writes
+Qcache_hits
+Qcache_inserts
+Qcache_lowmem_prunes
+Qcache_not_cached
+Queries
+Questions
+Select_full_join
+Select_full_range_join
+Select_range
+Select_range_check
+Select_scan
+Slow_launch_threads
+Slow_queries
+Sort_merge_passes
+Sort_range
+Sort_rows
+Sort_scan
+Table_locks_immediate
+Table_locks_waited
+Table_open_cache_hits
+Table_open_cache_misses
+Table_open_cache_overflows
+Tc_log_page_waits
+)
+
+intervals = interval_keys.map do |key|
+  [key, (status[key].to_i - old_status[key].to_i)]
+end.to_h
```

Comment:
`nil.to_i` is `0`, so any values not available will be marked as `0`. For now for the general case, I think a `0`-valued metric is close enough to a `nil` (not-reported) metric that we don't need to care too much about collapsing the difference. (and we can revisit if a specific case/issue arises)

The initial-interval data won't be recorded (thanks to the `exit 0 unless status[counter] && old_status[counter]` line). This could be optimized to report just the `constant` values (and not the `interval` values) on the initial run, but not necessary for the first pass at capturing these metrics for now.